avidemux_ru.ts

```diff
@@ -2881,7 +2881,7 @@ Is it running ?</source>
         <location line="+8"/>
         <location line="+2"/>
         <source>Clear recent items</source>
-        <translation>Очистить список</translation>
+        <translation>Очистить список недавних</translation>
```

Comment:
Substantive missing? Sounds very broken for me. I believe the previous translation was better.

Reply:
Okey, I'll revert it.